src/flag_gems/utils/pointwise_dynamic.py

```diff
@@ -392,9 +397,18 @@ def generate_destination_passing_pointwise_wrapper(
         if rank > 0:
             code.writeline("# strides of each tensor argument w.r.t the task space")
             for i in range(op_desc.num_input_tensors()):
-                code.writeline(
-                    f"in{i}_strides = broadcasted_stride(in{i}.shape, in{i}.stride(), shape)"
-                )
+                code.writeline(f"if 'in{i}_shape' in kwargs:")
```

Comment:
This modification relax the requirements that operand's shape are broadcastable when explicitly passed in input shape.